index.css

```diff
@@ -0,0 +1,371 @@
+}
+.team__title,.footer__title{
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 5px;                                    
+    letter-spacing: 0px;
+    margin: 0;
+    margin-top: 39px;
+    padding: 0;
+    color: #252B42;
+}
+
+.team__desc,.footer__decs{
+    font-weight: 400;
+    font-size: 14px;
+    line-height: 20px;
+    letter-spacing: 0.2px;
+    color: #737373;
+    margin-top: 30px;
+}
+.card{
+    display: grid;
+    text-align: center;
+    grid-template-columns: auto auto auto auto;
+    width: fit-content;
+    gap: 20px;
+    margin: auto;
+    margin-top: 90px;
+}
+.card__item{
+    border-radius: 14px;
+    -webkit-box-shadow: 0px 0px 5px 0px rgba(0, 0, 0, 0.07);
+    -moz-box-shadow: 0px 0px 5px 0px rgba(0, 0, 0, 0.07);
+    box-shadow: 0px 0px 5px 0px rgba(0, 0, 0, 0.07);
+}
+.card__img{
+    object-fit: cover;
+    width: fit-content;
+    border-top-left-radius: 14px;
+    border-top-right-radius: 14px;
+}
+.card__footer{
+
+    text-align: left;
+    width: fit-content;
+    display: flex;
+    margin-left: -5px;
+}
+.card__icons{
+    text-align: left;
+    display: inline;
+    margin-left: 20px;
+    width: fit-content;
+    margin-left: 25px;
+}
+.card_svg{
+    width: 25px;
+    margin: 0;
+    padding: 0;
+}
+.card__title{
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 24px;
+    text-align: center;
+    letter-spacing: 0.1px;
+    color: #252B42;
+    margin: 0;
+    padding: 0;
+    margin-top: 20px;
+}
+.card__desc{
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 16px;
+    text-align: center;
+    letter-spacing: 0.2px;
+    color: #737373;
+    margin: 0;
+    padding: 0;
+    margin-top: 10px;
+}
+.footer{
+    padding-left: 10%;
+    padding-right: 5%;
+    padding-top: 80px;
+    padding-bottom: 60px;
+    text-align: center;
+}
+.footer__smalltitle{
+    font-style: normal;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 14px;
+    color: #1BA0AB;
+}
+.form{
+    display: flex;
+    text-align: center;
+    margin: auto;
+    width: 50%;
+}
+.form__input{
+    display: flex;
+    text-align: center;
+
+}
+.form__item--input{
+    width: 100%;
+}
+.form__input{
+    background: #F9F9F9;
+    border: 1px solid #E6E6E6;
+    border-radius: 5px;
+    padding: 12px;
+    width: 100%;
+    outline: none;
+    text-align: left;
+
+}
+.form__input::placeholder{
+    font-weight: 500;
+    font-family: 'Montserrat', sans-serif ; 
+}
+.form{
+    margin-top: 60px;
+    margin-bottom: 20px;
+}
+.form__item--right{
+    background: #1BA0AB;
+    border: 1px solid #E6E6E6;
+    color: white;
+    border-radius: 0px 5px 5px 0px;
+    width: 117px;
+    font-family: 'Montserrat', sans-serif ;
+}
+@media only screen and (max-width:1151px) {
+    .card{
+        grid-template-columns: auto auto auto;
+    }
+}
+@media only screen and (max-width:884px) {
+    .card{
+        grid-template-columns: auto auto;
+    }
+}
+@media only screen and (max-width:884px) {
```

Comment:
why did you duplicate `@media only screen and (max-width:884px) ` you already have it